ts/queue.ts

```diff
@@ -38,7 +38,7 @@ export default class Queue {
       expires,
       deadLetterExchange,
       deadLetterRoutingKey,
-      maxLength
+      maxLength,
```

Comment:
```suggestion
      maxLength,
      arguments
```

Reply:
if I'm not wrong a strict mode does allow us to create a variable name with arguments as it is predefined in javascript. it gives me an error that's why I have not destructed on L42.

Reply:
Oh yes, makes sense